data/json/flags.json

```diff
@@ -804,6 +804,24 @@
     "//": "Prevents the item from making the body part count as unfriendly to water and thus causing negative morale from being wet.",
     "info": "This clothing <good>performs well</good> even when <info>soaking wet</info>.  This can feel good."
   },
+  {
+    "id": "ITEM_BROKEN",
+    "type": "json_flag",
+    "context": [  ],
+    "info": "This item was broken and <bad>won't activate anymore</bad>."
```

Comment:
I feel that perhaps this should explain that it was broken specifically by water, otherwise it might be rather confusing.  Although if you do make that change then the flag name should probably change to indicate that it refers to specifically that sort of brokenness.

Reply:
If the item was broken by water, it should have `WATER_BREAK` flag explaining that it was probably broken by water. Leaving it like this allows others to use this flag when they want a item to be broken.

Reply:
For example, we can extend it to EMPs.

Reply:
EMP should only break items with modern transistors afaik, so it's not that simple.
Add another flag EMP_RESIST to things like electric firestarter?

Reply:
The behavior would be to flag items that would be broken by an EMP, and then apply this falg to them.